website/docs/r/security_center_auto_provisioning.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_auto_provisioning"
+description: |-
+  Manages the subscription's Security Center Auto Provisioning.
+---
+
+# azurerm_security_center_auto_provisioning
+
+Enables or disables the Security Center Auto Provisioning feature for the subscription
+
+~> **NOTE:** There is no resource name required, it will always be "default"
+
+## Example Usage
+
+```hcl
+resource "azurerm_security_center_auto_provisioning" "example" {
+  auto_provision = "On"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `auto_provision` - (Required) Either "On" or "Off" to enable or disable Auto Provisioning. This is not case sensitive.
```

Comment:
the Terraform field for this is case-sensitive, so I think this can become:

```
* `auto_provision` - (Required) Either "On" or "Off" to enable or disable Auto Provisioning.
```

However maybe it's worth explaining a little more what this does:

```suggestion
* `auto_provision` - (Required) Should the security agent be automatically provisioned on Virtual Machines in this subscription? Possible values are `On` (to install the security agent automatically, if it's missing) or `Off` (to not install the security agent automatically).
```